custom_components/frigate/api.py

```diff
@@ -31,9 +30,12 @@ class FrigateApiClientError(Exception):
 class FrigateApiClient:
     """Frigate API client."""
 
-    def __init__(self, host: str, session: aiohttp.ClientSession) -> None:
+    def __init__(
+        self, host: str, timezone: str, session: aiohttp.ClientSession
```

Comment:
If you end up keeping timezone as state in this class, suggest moving it to the end of the constructor and defaulting to UTC so that you don't need to specify it in many places (e.g. unittests).